aspnetcore/tutorials/build-your-first-razor-components-app.md

```diff
@@ -143,7 +153,7 @@ Add a new page to the app that implements a simple todo list.
 
 1. Add a *TodoItem.cs* file to the root of the project to hold a class that represents a todo item. Use the following C# code for the `TodoItem` class:
 
-   [!code-cshtml[](build-your-first-razor-components-app/samples/3.x/WebApplication1/WebApplication1.App/TodoItem.cs)]
+   [!code-cshtml[](build-your-first-razor-components-app/samples/3.x/RazorComponents/RazorComponents.App/TodoItem.cs)]
 
 1. Return to the Todo component (*Todo.cshtml*):
```

Comment:
The highlighting in the code snippet for this section looks a bit off.

Reply:
I double-checked locally, and it looks :eye: ok. I can't see the doc's system build (internal review topic), but I'll find out quickly right after it goes live what's what. I can do a quick patch if anything breaks. Locally, these highlights seem to be ok.